app/styles/app.scss

```diff
@@ -87,3 +87,22 @@ body {
 .btn btn-danger, btn-primary, a, hover {
   color: $white;
 }
+
+.item{
+    background: #333;    
+    text-align: center;
+    height: 300px !important;
+}
+.carousel{
+    margin-top: 20px;
+}
+.bs-example{
+  margin: 20px;
+}
+h2{
+    margin: 0;     
+    color: #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces